lib/theme/text_theme.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'colors.dart';
+
+/// The referenced design (Figma) is based on the Unity Style Typography. But
+/// the one we are supposed to follow is Material Design Typography. Each [TextStyle]
+/// defined below corresponds to the matching text style defined in Unity.
+const TextTheme textTheme = TextTheme(
+  /// Body/B1
+  bodyLarge: TextStyle(
+    color: AllAppColors.label,
+    fontSize: 16,
+    letterSpacing: 0.2,
+  ),
+
+  /// Body/B2
+  bodyMedium: TextStyle(
+    color: AllAppColors.label,
+    fontSize: 14,
+    letterSpacing: 0.22
+  ),
+
+  /// Button/BT2
+  labelLarge: TextStyle(
+    color: AllAppColors.label,
+    fontWeight: FontWeight.bold,
+    fontSize: 14,
+    letterSpacing: 0.32
+  ),
```

Comment:
There is no line height mentioned. Any reason why?

Reply:
No Rahul. Line heights are in fact important here. It's my mistake that I missed out on defining line heights of the TextStyles.